bin/common.sh

```diff
@@ -105,4 +105,23 @@ export_mem_limit_from_conf() {
 
     export TCMALLOC_HEAP_LIMIT_MB=${mem}
     return 0
-}
\ No newline at end of file
+}
+
+export_shared_envvars() {
+    # ===================================================================================
+    # initialization of environment variables before exporting env variables from be.conf
+    # For most cases, you should put default environment variables in this section.
+    #
+    # UDF_RUNTIME_DIR
+    # LOG_DIR
+    # PID_DIR
+    export UDF_RUNTIME_DIR=${STARROCKS_HOME}/lib/udf-runtime
+    export LOG_DIR=${STARROCKS_HOME}/log
+    export PID_DIR=`cd "$curdir"; pwd`
+
```

Comment:
Why not move `DORIS_HOME` here too?

Reply:
OK. Fixed, move `$DORIS_HOME` to this function too.